src/schema/workflow_execution_activity.yaml

```diff
@@ -307,7 +314,11 @@ classes:
         structured_pattern:
           syntax: "{id_nmdc_prefix}:(omprc|dgms)-{id_shoulder}-{id_blade}$"
           interpolated: true
-
+      uses_calibration:
+        structured_pattern:
+          syntax: "{id_nmdc_prefix}:calib-{id_shoulder}-{id_blade}$"
+          interpolated: true
+     
```

Comment:
If you move the structured_pattern to the global slot def you can remove this slot usage altogether.